Examples/Swift/UserTrackingModesExample.swift

```diff
@@ -4,7 +4,6 @@ import Mapbox
 
 class UserTrackingModesExample_Swift: UIViewController, MGLMapViewDelegate {
     var mapView: MGLMapView!
-    @IBOutlet var button: UserLocationButton!
```

Comment:
Can this property also be removed from the Objective-C example? https://github.com/mapbox/ios-sdk-examples/blob/master/Examples/ObjectiveC/UserTrackingModesExample.m#L112

Reply:
→ 152c49